lib/xeroizer/models/credit_note.rb

```diff
@@ -126,8 +126,26 @@ def total(always_summary = false)
         def pdf(filename = nil)
           parent.pdf(id, filename)
         end
-              
+
+        def save
+          super
+          allocate if !allocations.empty?
```

Comment:
@CloCkWeRX Why `dup`? It looks like calling `save` on the credit note won't edit the `allocations` array; it just replaces it wholesale during `parse_save_response`. So unless I'm missing something, I don't see a reason to duplicate it when we back it up.